frontend/src/stores/userStore.ts

```diff
@@ -59,7 +57,7 @@ export const useUserStore = defineStore('users', {
 		async refreshAllUsers() {
 			try {
 				const data = await getRequest('users');
-				this.allUsers = data.data;
+				this.allUsers = data.data.filter((user: User) => (user.id !== this.me.id));
```

Comment:
I understand now that this is to view "all users" on the /profiles page

Point to discuss:
also show the logged in user their profile
this way a user can see their own profile in the same way it would also be visible to others

Reply:
Uuummmmm apples :D

See above comment about `view my profile`.